app-emulation/rkt/rkt-9999.ebuild

```diff
@@ -28,30 +30,42 @@ IUSE=""
 
 DEPEND=">=dev-lang/go-1.3
 	app-arch/cpio
-	sys-fs/squashfs-tools"
+	sys-fs/squashfs-tools
+	dev-perl/Capture-Tiny"
```

Comment:
This will need to be added to coreos-base/sdk-depends in order to include in the SDK. Sadly portage does not yet know when to build build dependencies into $ROOT and when to put them into /